README.md

```diff
@@ -75,6 +75,15 @@ GOOGLE_CLOUD_PROJECT=my-project
 GOOGLE_CLOUD_KEYFILE=/Users/user/.gcp/my-project/shrine-gcs-test.json
 ```
 
+Be sure you are setting the Google Cloud config using the variables just created.
+
+```sh
+Google::Cloud.configure do |config|
+    config.project_id = GOOGLE_CLOUD_PROJECT
+    config.keyfile = GOOGLE_CLOUD_KEYFILE
+end
+```
+
```

Comment:
I am not sure I understand this part. This section is about running the test suite of this Shrine plugin, not about how to use it.